### PR TITLE
Add exercises cards on clicking to exercises categories

### PR DESCRIPTION
**Pull request by olexandrd**

### CODE REVIEW

File: src/js/rerender.js [hunk 1]
```diff
@@ -26,4 +27,26 @@ function rerender() {
...
+
+      const activePage = Array.from(refs.pagination.childNodes).find(
+        p => p.dataset.id == data.page
+      );
+      if (activePage) {
+        activePage.classList.add('active');
+      }
+    })
+    .catch(error => {
+      console.log(error);
```
Comment (kazamov) [inline src/js/rerender.js]: In the list of requirements there is a point: no console logs. Or you need to show some alert/notification, or just ignore error

Comment (olexandrd) [reply src/js/rerender.js]: replaced with izitoast

File: src/js/eventHandlers.js [hunk 2]
```diff
@@ -13,8 +14,13 @@ function onPaginationClick(e) {
 
   const page = Number(paginationItem.dataset.id);
   queryParamsFilters.set('page', page);
+  queryParamsExercises.set('page', page);
 
-  rerender();
+  if (refs.exerciseHeader.dataset.exercise) {
+    rerenderExercises(queryParamsExercises);
```
Comment (Sysqwerty) [inline src/js/eventHandlers.js]: There is no longer any need to pass arguments, the service serviceGetExercises no longer accepts parameters.

Comment (olexandrd) [reply src/js/eventHandlers.js]: removed

**Review by Sysqwerty — DISMISSED** [review]

**Review by Sysqwerty — APPROVED** [review]

**Pull request merged**